pkg/goutil/goutil_test.go

```diff
@@ -0,0 +1,19 @@
+package goutil
+
+import "testing"
+
+func TestInt32Addr(t *testing.T) {
+	tests := []struct {
+		input  int32
+		output *int32
+	}{
+		{1, Int32Addr(1)},
+		{2, Int32Addr(2)},
+	}
+
+	for _, test := range tests {
+		if test.input != *test.output {
+			t.Errorf("The input and output doesn't match, for %d", test.input)
```

Comment:
logically you should print both of those values to know what's wrong, not just input